src/DynamoCoreWpf/Controls/IncanvasSearchControl.xaml

```diff
@@ -23,8 +23,8 @@
             <Style x:Key="SearchMemberStyle"
                    BasedOn="{StaticResource MemberGroupMemberStyle}"
                    TargetType="{x:Type ListBoxItem}">
-                <EventSetter Event="PreviewMouseLeftButtonDown"
-                             Handler="OnMouseLeftButtonDown" />
+                <EventSetter Event="PreviewMouseLeftButtonUp"
+                             Handler="OnMouseLeftButtonUp" />
```

Comment:
I remember seeing this change for the drag-and-drop PR two weeks ago, hopefully this change doesn't break that (not sure why is this changed here).